### PR TITLE
Ui

### PR DESCRIPTION
**Pull request by nickbalestra**
- Cleaned UI, addressing #13.
- Removed initial hardcoded query

### CODE REVIEW

File: src/static/template.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <meta name="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="author" content="Nick Balestra && Mattia Richetto"/>
+    <link rel="stylesheet" href="http://yui.yahooapis.com/pure/0.6.0/pure-min.css">
```
Comment (mattiaerre) [inline src/static/template.html]: this is really cool; I didn't know about this project I ❤️ it!

Comment (mattiaerre) [reply src/static/template.html]: can we `npm` it? or you reckon is better as it is through its CDN?

File: src/app.jsx [hunk 2]
```diff
@@ -56,35 +56,52 @@ export default class App extends React.Component {
...
+
+              <div className="query-content-editor-controls pure-u-1-6">
+                <button className="primary-button pure-button" onClick={this.handleOnClick}>{this.state.labels.button}</button>
+              </div>
+            </div>
+
+
+            <div className="query-content-result">
+              <pre>{JSON.stringify(this.state.response.json, null, 2) }</pre>
+              {/*<pre>{JSON.stringify(this.state.error, null, 2) }</pre>*/}
```
Comment (mattiaerre) [inline src/app.jsx]: what happens when we have an error? is this still a WIP?

File: src/app.jsx [hunk 3]
```diff
@@ -9,9 +9,9 @@ export default class App extends React.Component {
 
     this.state = {
       labels: {
-        button: 'get from query',
+        button: 'Get',
       },
-      query: '["metrosById", [72], ["name"]]',
+      query: 'Your query here',
```
Comment (mattiaerre) [inline src/app.jsx]: what about **Enter Falcor query** a la postman? 😅 

screen shot 2016-10-06 at 21 29 38

**Review by mattiaerre — COMMENTED** [review]
awesome @nickbalestra over to you!
btw I'd love to add also a header to the UI

File: src/static/template.html (hunk 1 above)
Comment (nickbalestra) [reply src/static/template.html]: yup, good idea, lets `npm i` it

File: src/app.jsx (hunk 3 above)
Comment (nickbalestra) [reply src/app.jsx]: Sweet! 💅

Comment (nickbalestra) [reply src/app.jsx]: I also think we should move that default to be holded somewhere else in the state, and not in the query.

File: src/app.jsx (hunk 2 above)
Comment (nickbalestra) [reply src/app.jsx]: Fixed, we now handle both in the same output.

**Comment by nickbalestra** [comment]
@mattiaerre done, over to you.
For the Header, agree is a good idea, let's open an issue for it, so we can discuss there next features and how to evolve the UI accordingly

File: src/app.jsx (hunk 3 above)
Comment (mattiaerre) [reply src/app.jsx]: Agree about that; it looks like a "placeholder" kind of attribute right?

File: src/app.jsx (hunk 2 above)
Comment (mattiaerre) [reply src/app.jsx]: 👍

File: package.json [hunk 4]
```diff
@@ -70,6 +69,7 @@
     "nodemon": "^1.10.2",
     "postcss-loader": "^0.13.0",
     "pre-commit": "^1.1.3",
+    "purecss": "^0.6.0",
     "react": "^15.3.1",
     "react-codemirror": "^0.2.6",
     "react-dom": "^15.3.1",
```
Comment (mattiaerre) [inline package.json:74]: quick one, are we really using all these dependencies? shall we run an `npm-check` just in case?

**Review by mattiaerre — COMMENTED** [review]
all LGTM 👍 let me know if I can merge it!

File: package.json (hunk 4 above)
Comment (nickbalestra) [reply package.json:74]: Cleaned it. I think it should be pretty ok now. Good to merge i think.

**Comment by mattiaerre** [comment]
LGTM 👍

**Pull request merged**